pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -82,11 +82,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for changes to DirectImageStreamMigrations
-	err = c.Watch(&source.Kind{Type: &migapi.DirectImageStreamMigration{}},
-		&handler.EnqueueRequestForOwner{
-			IsController: true,
-			OwnerType:    &migapi.DirectImageMigration{},
-		}, &migref.MigrationNamespacePredicate{Namespace: migapi.OpenshiftMigrationNamespace})
+	err = c.Watch(
+		source.Kind(mgr.GetCache(), &migapi.DirectImageStreamMigration{}),
+		handler.EnqueueRequestForOwner(
+			mgr.GetScheme(),
+			mgr.GetRESTMapper(),
```

Comment:
Not sure if it matters, I did mgr.GetClient().RESTMapper(). They are probably the same thing.